.github/workflows/gradle.yml

```diff
@@ -9,25 +9,30 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    permissions:
+      checks: write
+      contents: read
```

Comment:
These permissions are needed for the `scacap/action-surefire-report` action to work properly